subprojects/configuration-cache/src/main/kotlin/org/gradle/configurationcache/serialization/codecs/transform/Transforms.kt

```diff
@@ -84,7 +84,7 @@ fun unpackTransformationStep(node: TransformationNode): TransformStepSpec {
 
 
 fun unpackTransformationStep(transformation: TransformationStep, upstreamDependencies: TransformUpstreamDependencies): TransformStepSpec {
-    return if (transformation.requiresDependencies()) {
+    return if (transformation.transformer.requiresDependencies()) {
```

Comment:
💄 Consider introducing a `transformation.requiresDependencies()` method to minimize changes (and appease Demeter).

Reply:
Done.